pkg/lib/cockpit-components-listing.jsx

```diff
@@ -64,8 +67,8 @@ var ListingRow = React.createClass({
     },
     getInitialState: function() {
         return {
-            expanded: false, // show expanded view if true, otherwise one line compact
-            activeTab: 0,    // currently active tab in expanded mode, defaults to first tab
+            expanded: this.props.initiallyExpanded, // show expanded view if true, otherwise one line compact
+            activeTab: this.props.initiallyActiveTab == null ? 0 : this.props.initiallyActiveTab,    // currently active tab in expanded mode, defaults to first tab
```

Comment:
is null actually different from undefined? If so this needs to be === otherwise just ```this.props.initiallyActiveTab ? this.props.initiallyActiveTab : 0``` should be good.